### PR TITLE
Switch to using o-overlay on npm instead of bower

### PR DESCRIPTION
**Pull request by remybach**
Membership are trying to implement this on https://github.com/Financial-Times/m-login, but because that's not built on the Next stack they're having difficulties doing so. One of which being that they don't have bower set up in the workflow _at all_.

Since Origami components are available on npm as well, it's much easier to just do that here.

🐿 v2.12.5

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
   "main": "index.js",
   "dependencies": {
     "n-ui-foundations": "^3.0.0",
-    "o-overlay": "^2.7.5",
```
Comment (liamkeaton) [inline bower.json]: Are there styles within `o-overlay` which may still need this to be a bower dependency?

Comment (remybach) [reply bower.json]: I tested this locally on `next-profile` and it seemed to work, but I'm not sure what this will do on `m-login`.

Comment (remybach) [reply bower.json]: :( It probably worked because my `bower_components` was still installed.

Comment (liamkeaton) [reply bower.json]: :(

**Comment by remybach** [comment]
Yep, this isn't going to work.

**Comment by liamkeaton** [comment]
spock-sorry

**Pull request closed without merging**